src/components/atoms/FormRadioInput.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="radio-input">
+    <input
+     class="radio-value"
+      type="radio"
+      v-bind="$attrs"
+      :checked="modelValue === value"
+      :value="value"
+      @change="$emit('update:modelValue', value)"
+    />
+    <label class="radio-label" v-if="label">{{ label }}</label>
```

Comment:
phần này anh thêm for vào để click vào chứ nó chọn luôn radio button cho mình nhé, trải nghiệm người dùng tốt hơn